Classes/util/ColorJSONConverter.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
```

Comment:
I would rather name the folder utils instead of util as we might need to add other utils 'til the end of the project